docs/enterprise/troubleshooting.md

```diff
@@ -1,6 +1,7 @@
 ---
-title: Troubleshooting lakeFS Enterprise
+title: Troubleshooting
```

Comment:
I think the previous title, while not as succinct, makes it easier to find in the search bar

Reply:
done